### DISCUSSION TITLE
Are you Planning to add Media Library ?

### DISCUSSION THREAD

**Discussion by manishonc** [pocketbase/pocketbase#1288] [Q&A] [answered]
I am struggling to add files I have to upload same file again n again.

If there is a media Library from where I can choose media file.
That would save us lots of time and Save our storage.

Also are you planning to Add any Text Editor 
Like CKeditor

So we can directly Directly write post in Pocketbase admin panel
?

Thanks for making pocketbase 
It's one of a kind and amazing .

**Comment by ganigeorgiev** [comment, marked as answer]
> Are you Planning to add Media Library?

No, but you can achieve the same behavior on your own by creating a dedicated collection (eg. `media`) and use the `relation` type field to point to it in your other collections.

> Also are you planning to Add any Text Editor

It is in the [roadmap](https://github.com/orgs/pocketbase/projects/2). You can check https://github.com/pocketbase/pocketbase/issues/370#issuecomment-1355531055.

**Reply by manishonc** [reply]
Thanks That's Good idea
